src/shared/services/activity/activity.service.ts

```diff
@@ -47,23 +47,34 @@ export class ActivityService {
   }
 
   public getActivities(...caseId: string[]): Observable<Activity[]> {
-    const options = this.getOptions();
-    const url = this.activityUrl() + `/cases/${caseId.join(',')}/activity`;
-    return this.http
-      .get(url, options, false, ActivityService.handleHttpError)
-      .map(response => response);
+    try {
+      const options = this.getOptions();
+      const url = this.activityUrl() + `/cases/${caseId.join(',')}/activity`;
+      return this.http
+        .get(url, options, false, ActivityService.handleHttpError)
+        .map(response => response);
```

Comment:
What is the use of the map? We are not transforming the response at all. Please remove!

Reply:
for time being I can't remove it as it is already in production, but Uday has mentioned some changes yesterday so when we implement that change we can get rid of this.